internal/pkg/peer/orderers/connection.go

```diff
@@ -81,6 +82,20 @@ func (cs *ConnectionSource) Endpoints() []*Endpoint {
 	return cs.allEndpoints
 }
 
+// ShuffledEndpoints returns a shuffled array of endpoints in a new slice.
```

Comment:
You added this method but now no one calls `Endpoints()` anymore. 
Maybe delete `Endpoints()` ?